src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java

```diff
@@ -135,6 +135,63 @@ void testCreateFinancialTransaction_WhenDescriptionIsEmpty_ThenCreateFinancialTr
                 .mapFinancialTransactionEntityToFinancialTransactionDTO(any());
     }
 
+    @Test
+    @DisplayName("do not update financial transaction without valid id and throw AppRuntimeException")
+    void shouldThrowExceptionWhenUpdatingFinancialTransactionWithInvalidId() {
+        //given
+        FinancialTransactionUpdateDTO updateDTO = createFinancialTransactionUpdateDTO();
+        when(financialTransactionRepository.findById(any())).thenReturn(Optional.empty());
+
+        //when & then
+        AppRuntimeException exception = assertThrows(
+                AppRuntimeException.class,
+                () -> financialTransactionService.updateFinancialTransaction(ID_1L, updateDTO)
+        );
+        assertEquals(ErrorCode.FT001.getBusinessStatusCode(), exception.getBusinessStatusCode());
+        assertEquals(ErrorCode.FT001.getBusinessMessage(), exception.getBusinessMessage());
```

Comment:
You can add one empty line before verifying the block code - just like you did in `shouldUpdateFinancialTransactionWhenValidParametersAreGiven()` test method.